examples/set-namespace-depends-on/resources.yaml

```diff
@@ -3,6 +3,7 @@ kind: Deployment
 metadata:
   annotations:
     config.kubernetes.io/depends-on: apps/namespaces/default/StatefulSet/wordpress-mysql
+    internal.kpt.dev/upstream-identifier: apps|Deployment|example|wordpress
```

Comment:
When will this annotation be set? When we `kpt pkg get` a package?

Reply:
kpt CLI will add this annotation in `kpt pkg get ` and `kpt pkg udpate`. porch will add this annotation in clone and pkg update.